.github/workflows/reusable-ubuntu.yml

```diff
@@ -4,9 +4,11 @@ on:
       config_hash:
         required: true
         type: string
-      options:
-        required: true
-        type: string
+      free-threading:
+        description: Whether to use no-GIL mode
```

Comment:
Let's avoid the forbidden term :)
```suggestion
        description: Whether to use free-threaded mode
```

Reply:
It's forbidden? :o

Reply:
Yes, see https://github.com/python/devguide/issues/1353.

Reply:
Thanks for the reminder, I missed it here.

Reply:
@hugovk FYI the buildbots still use no-GIL terminology: https://buildbot.python.org/#/builders/1258/builds/2403